phenol-analysis/src/test/java/org/monarchinitiative/phenol/analysis/scoredist/H2ScoreDistributionReaderTest.java

```diff
@@ -32,8 +36,9 @@ public void setUp() throws Exception {
     objDbReader = new H2ScoreDistributionReader(dbpath.toFile().getAbsolutePath(), dataTableName);
 
     TermId termId = TermId.of("MONDO", "001");
-    ObjectScoreDistribution scoreDistribution = new ObjectScoreDistribution(termId, 2, 3, ImmutableSortedMap.of(0.0, 0.2, 0.5, 0.6, 1.0, 1.0));
-    ImmutableMap<TermId, ObjectScoreDistribution> scoreDistributionMap = ImmutableMap.of(termId, scoreDistribution);
+    SortedMap<Double, Double> cumulativeFrequencies = new TreeMap<>(Map.of(0.0, 0.2, 0.5, 0.6, 1.0, 1.0));
```

Comment:
Don't use `SortedMap` unless you've thoroughly tested the functionality. We used this in svart in the GenomicAssemblies and the behaviour is not the same as that seen in `ImmutableSortedMap` see https://github.com/exomiser/svart/commit/8834205a93fdeae7e6a4db0614a3a0ca045741c4

Reply:
This is a good note, thanks for letting me know. I missed this subtlety in svart, and I will double check the `hashCode` implementation of the key everytime I use `SortedMap`.

It should not be an issue in this particular case though, should it? `Double` uses `Double.toLongBits()` when performing equality test, and this should always be OK, unless we're super freaked out by machine-dependent imprecisions.

Reply:
Write a tiny unit test, check the behaviour is as you expect and then adapt accordingly. I think it will be fine, however.